hw2/templates/hello.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %} {% block content %}
+
+<h1 class="title">Добро пожаловать</h1>
+
+<form action="/logout/">
```

Comment:
Тогда требуют `action` не понадобится, но у вас почему-то не указан тип запроса.
```suggestion
<form method='post'>
```